### ISSUE TITLE
When Firefox has been updated, the url's don't resolve 

### ISSUE THREAD

**Issue by HIndeqa** [null-dev/firefox-profile-switcher#54]
When Firefox has been updated, new or open tabs are not loaded. When you try to open an url, nothing happens. You have to close all the profiles and restart Firefox to make this happen. This is very annoying when you have 5 profiles open, especially when it happens after every FF update

**Comment by null-dev** [comment]
Are you sure this is an issue with this extension? I thought this was just how Firefox updated because I see the same issues on my systems that don't have this extension installed.

**Comment by HIndeqa** [comment]
Thank you for your response, that is a good question. I use FF privately with a lot of extensions, but not with Profile Switcher. For business I use only 2 other extensions (adblocker and a password manager). I don't have the issue privately, only at work.

It is possible that the issue only occurs when I open a second profile, I'm not sure about that. I can confirm that when FF has been updated again

**Comment by johnmcdanny** [comment]
Yeah I've got the same issue here. Only happens when FF is updated and only the window using the default profile can't open new urls. To circumvent this I need to:
1) close the window running the default profile (no need to close the other windows)
2) open FF again (either from another profile using profile switcher or by launching FF normally)
3) press "close FF" on the "FF is already running popup"
4) launch FF again

**Comment by HIndeqa** [comment]
Firefox just updated and I think I know what happens. The first instance opens and checks if there is an update. Then the second one opens and applies the update. Then the first one stops working because it thinks it's not update. Something like that. It might indeed be a Firefox issue which comes to light because of the profile switcher
